pnpm-workspace.yaml

```
@@ -1,4 +1,5 @@
 packages:
   - 'js'
   - 'expo-plugin-nogodey'
-  - 'examples/*' 
\ No newline at end of file
+  - 'examples/*' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)